src/controllers/auth/forgotPassword.ts

```diff
@@ -45,8 +49,26 @@ export const postForgot = asyncHandler(
       throw new InternalServerError()
     }
 
-    // TODO: email the token to the user/guest, return a 204 response
-    logger.info({ token })
-    throw new NotImplemented()
+    try {
+      const urlPath = urljoin(
+        <string>env(EnvVariable.APP_URL, 'http://localhost:3000'),
+        <string>env(EnvVariable.AUTH_PASSWORD_RESET_PATH, '/password/reset'),
+      )
```

Comment:
This is using a type assertion, please update this to pass `string` as a generic. These are two different things. 

```suggestion
      const urlPath = urljoin(
        env<string>(EnvVariable.APP_URL, 'http://localhost:3000'),
        env<string>(EnvVariable.AUTH_PASSWORD_RESET_PATH, '/password/reset'),
      )
```

Reply:
Complete